Commands/Commands/uservoice.js

```diff
@@ -1,10 +1,10 @@
-var commands = []
+let commands = []
 
-// var checker = require('../../Utils/access_checker')
-var logger = require('../../Utils/error_loggers')
-var config = require('../../config.js')
-var Entities = require('html-entities').AllHtmlEntities
-var UVRegex = /https?:\/\/[\w.]+\/forums\/(\d{6,})-[\w-]+\/suggestions\/(\d{7,})(?:-[\w-]*)?/
+// const checker = require('../../Utils/access_checker')
```

Comment:
If your moving variables like these over to ``const``'s, doesn't that make those variable program wide? So why repeat them, and as ``const`` after moving them to that?